tests/autoyast/installation.pm

```diff
@@ -116,7 +116,7 @@ sub verify_timeout_and_check_screen {
 sub run {
     my ($self) = @_;
 
-    if (check_var('BACKEND', 'ipmi') && get_var("SES5_DEPLOY")) {
+    if (is_remote_installation()) {
```

Comment:
careful, there is a '&&' here, not '||'

Reply:
Then now I am confused with this one.
I have to talk to people from storage team.